lib/vanagon/platform/windows.rb

```diff
@@ -55,6 +55,7 @@ def generate_packaging_artifacts(workdir, name, binding)
       def generate_msi_packaging_artifacts(workdir, name, binding)
         FileUtils.mkdir_p(File.join(workdir, "wix"))
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service.component.wxs.erb"), File.join(workdir, "wix", "service.#{name}.wxs"), false, { :binding => binding })
+        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.service-filter.xslt.erb"), File.join(workdir, "wix", "#{name}.service-filter.xslt"), false, { :binding => binding })
```

Comment:
Did we decide to leave this as `filter` rather then `service-filter` to deal with the potential we may at some point need to filter things not associated with the services?